exampleCourse/courseInstances/Sp15/assessments/plelementexamples/infoAssessment.json

```diff
@@ -35,9 +35,10 @@
             "questions": [
 				{"id": "examplesCheckbox",           "points": 2, "maxPoints": 10},
 				{"id": "examplesStringInput",           "points": 2, "maxPoints": 10},
-				{"id": "elementsMatrixLatexDisplay",        "points": 2, "maxPoints": 10},
+		{"id": "elementsMatrixLatexDisplay",        "points": 2, "maxPoints": 10},
                 {"id": "examplesMatrixInput",        "points": 2, "maxPoints": 10},
-                {"id": "examplesSymbolicInput",        "points": 2, "maxPoints": 10}
+                {"id": "examplesSymbolicInput",        "points": 2, "maxPoints": 10},
```

Comment:
Can you tidy this chunk up to use spaces (the first 2 questions) and a consistent indent?